ThirdPartyAdapters/liftoffmonetize/liftoffmonetize/src/main/java/com/google/ads/mediation/vungle/renderers/VungleAppOpenAd.kt

```diff
@@ -75,33 +73,17 @@ abstract class VungleAppOpenAd(
 
     val context: Context = mediationAppOpenAdConfiguration.context
 
-    VungleInitializer.getInstance()
-      .initialize(
-        // Safe to access appId here since we do a null-check for appId earlier in the function and
-        // return if it's null.
-        appId!!,
-        context,
-        object : VungleInitializationListener {
-          override fun onInitializeSuccess() {
-            val adConfig = vungleFactory.createAdConfig()
-            if (mediationExtras.containsKey(VungleConstants.KEY_ORIENTATION)) {
-              adConfig.adOrientation =
-                mediationExtras.getInt(VungleConstants.KEY_ORIENTATION, AdConfig.AUTO_ROTATE)
-            }
-            maybeAddWatermarkToVungleAdConfig(adConfig, mediationAppOpenAdConfiguration)
-            // Note: Safe to access placement here since we do a null-check for placement earlier in
-            // the function and return if it's null.
-            appOpenAd = vungleFactory.createInterstitialAd(context, placement!!, adConfig)
-            appOpenAd.adListener = this@VungleAppOpenAd
-            appOpenAd.load(getAdMarkup(mediationAppOpenAdConfiguration))
-          }
-
-          override fun onInitializeError(error: AdError) {
-            Log.w(TAG, error.toString())
-            mediationAdLoadCallback.onFailure(error)
-          }
-        },
-      )
+    val adConfig = vungleFactory.createAdConfig()
+    if (mediationExtras.containsKey(VungleConstants.KEY_ORIENTATION)) {
+      adConfig.adOrientation =
+        mediationExtras.getInt(VungleConstants.KEY_ORIENTATION, AdConfig.AUTO_ROTATE)
+    }
+    maybeAddWatermarkToVungleAdConfig(adConfig, mediationAppOpenAdConfiguration)
+    // Note: Safe to access placement here since we do a null-check for placement earlier in
+    // the function and return if it's null.
+    appOpenAd = vungleFactory.createInterstitialAd(context, placement, adConfig)
```

Comment:
Same comment about the VungleFactory class 
![image](https://github.com/user-attachments/assets/0de9a947-f9ac-4f6c-b64e-e9d82d89895b)